code/modules/mob/living/carbon/combat.dm

```diff
@@ -23,3 +23,41 @@
 		else
 			return FALSE
 	return TRUE
+
+
+//Checks armor, special attackby of object instances, and miss chance
+/mob/living/carbon/proc/attacked_by(var/obj/item/I, var/mob/living/user, var/def_zone, var/originator = null)
+	. = 1
```

Comment:
There's no point to setting `.` here since every return explicitly states a value. Also use `TRUE`/`FALSE`

Reply:
Doesn't `.` also affect the value that is returned if the proc terminates without returning anything else first?
From what I can see, it is possible for the proc to reach its end without hitting a `return`, meaning this would see use.

Reply:
Yes give it a return true.